packages/manager/modules/telecom-spare/src/phones/order/phones-order.html

```diff
@@ -0,0 +1,212 @@
+<section class="telecom-legacy telecom-spare-phone-order-section">
+    <header>
+        <tuc-section-back-link data-tuc-section-back-link-to-state="spare.phones"
```

Comment:
Use a `oui-back-button` instead

Reply:
no the style is not the same than the other component of manager telecom. It's a QC demand  to have the same style.